guide/user-guide.rst

```diff
@@ -0,0 +1,4 @@
+User Guide
```

Comment:
Do not mix web user guide and developer guide. Use the same documentation layout as the other apps: 
* make a directory per user guide 
* introduction page of each guide is named index.rst